distribution/bin/enso

```diff
@@ -1,4 +1,4 @@
 COMP_PATH=$(dirname "$0")/../component
 JAVA_OPTS="--enable-native-access=org.graalvm.truffle --sun-misc-unsafe-memory-access=allow --add-opens=java.base/java.nio=ALL-UNNAMED"
-exec java --module-path $COMP_PATH $JAVA_OPTS -m org.enso.runner/org.enso.runner.Main "$@"
+exec $JAVA_HOME/bin/java --module-path $COMP_PATH $JAVA_OPTS -m org.enso.runner/org.enso.runner.Main "$@"
```

Comment:
- This is an API change!
- Luckily it doesn't affect production - production is using `ENSO_LAUNCHER=native` and not shell launchers
- So far we never used `JAVA_HOME` variable